enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/configuration/OnlineBackupSettings.java

```diff
@@ -38,6 +38,7 @@
     @Description( "Enable support for running online backups" )
     public static final Setting<Boolean> online_backup_enabled = setting( "dbms.backup.enabled", BOOLEAN, TRUE );
 
-    @Description( "Listening server for online backups" )
+    @Description( "Listening server for online backups. This will be the legacy backup protocol if setup is single instance or HA cluster. A copy of the " +
```

Comment:
I would phrase this a bit differently again, because the user is not generally aware of protocols. After the first sentence, I think it would be sufficient to remark that "backups can also be performed against the transaction port in a causal cluster" and cross-reference with ", see: causal_clustering.transaction_listen_address" which has more details regarding the network security.